public/protocols/development.netcanvas/protocol.json

```diff
@@ -1092,6 +1110,7 @@
         {
           "id": "6cl",
           "text": "Who would _you_ consider to be a **close friend**?",
+          "type": "eda5e3bb-8e1c-4216-9e06-adc0ff6b7f73",
```

Comment:
I don't think this one belongs; maybe just a bad copy/paste? If we delete it, the rest looks good.

Reply:
👍